assets/models/system/codellama-7b-instruct-hf/description.md

```diff
@@ -61,24 +68,22 @@ Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-text-generation" target="
 
 ### **Sample input**
 ```json
```

Comment:
inference_endpoint link is not correct. Please take reference from CodeLlama-13b-hf model

https://github.com/Azure/azureml-assets/blob/main/assets/models/system/CodeLlama-13b-hf/description.md